core/divelogexportlogic.cpp

```diff
@@ -148,8 +148,10 @@ void exportHtmlInitLogic(const QString &filename, struct htmlExportSetting &hes)
 	export_HTML(qPrintable(json_dive_data), qPrintable(photosDirectory), hes.selectedOnly, hes.listOnly);
 
 	QString searchPath = getSubsurfaceDataPath("theme");
-	if (searchPath.isEmpty())
+	if (searchPath.isEmpty()) {
+		report_error(qPrintable(gettextFromC::tr("Cannot find a folder called 'theme' in the standard locations")));
```

Comment:
If it is not user-facing, is it worth adding a translation string?

Reply:
it can be.

Reply:
If you think a new translation string  is warranted, I think we can commit this (after all @dirkhh gave his approval).